packages/mui-material/src/Checkbox/Checkbox.js

```diff
@@ -139,8 +142,22 @@ const Checkbox = React.forwardRef(function Checkbox(inProps, ref) {
 
   const classes = useUtilityClasses(ownerState);
 
+  const externalForwardedProps = {
+    ...other,
+    slots,
+    slotProps,
+  };
+
+  const [RootSlot, rootProps] = useSlot('root', {
```

Comment:
What's the runtime cost of this for the component?

Reply:
Added a task to measure it in https://github.com/mui/material-ui/issues/41281